shuup/front/apps/personal_order_history/dashboard_items.py

```diff
@@ -21,5 +21,9 @@ class OrderHistoryItem(DashboardItem):
 
     def get_context(self):
         context = super(OrderHistoryItem, self).get_context()
-        context["orders"] = Order.objects.filter(customer=self.request.customer).order_by("-created_on")[:5]
+        context["order_status_history"] = (
```

Comment:
no, keep showing the orders, just show the order history inside the order detail view